workflows/post_process_run/scream_post/format_output.py

```diff
@@ -0,0 +1,186 @@
+from typing import Hashable, List, Mapping
+import xarray as xr
+import numpy as np
+import os
+
+output_variables = [
+    "T_mid",
+    "qv",
+    "omega",
+    "horiz_winds",  # this will be changed to u and v in the future
+    "surf_sens_flux",
+    "surf_evap",
+    "SW_clrsky_flux_up@bot",
+    "SW_clrsky_flux_dn@bot",
+    "LW_clrsky_flux_dn@bot",
+    "LW_clrsky_flux_up@bot",
+    "SW_clrsky_flux_up@tom",
+    "LW_clrsky_flux_up@tom",
+    "VerticalLayerInterface",
+    "p_mid",
+    "ps",
+    "VerticalLayerInterface",
+    # "pseudo_density",  # this is pressure thickness
+    "area",
+    "lat",
+    "lon",
+    "hyai",
+    "hybi",
+]
+
+NP = 4
+NPG = 2
+levs = 128
+
+
+def make_placeholder_data(
+    sample: xr.DataArray, generate_variable: str, scaled_factor: float = 1.0
+):
+    placeholder = xr.DataArray(
+        np.random.rand(*sample.values.shape) * scaled_factor,
+        coords=sample.coords,
+        dims=sample.dims,
+    )
+    return placeholder.rename(generate_variable)
+
+
+def make_delp(p_mid, z_int, ps):
+    ptop = 225.52
+    delp = p_mid.copy()
+    dz = z_int[:, :, 0:-1] - z_int[:, :, 1:]
+    p_int = z_int.copy()
+    p_int[:, :, 0] = ptop
+    for k in range(1, levs):
+        p_int[:, :, k] = (
+            dz[:, :, k - 1] * p_mid[:, :, k] + dz[:, :, k] * p_mid[:, :, k - 1]
+        ) / (dz[:, :, k - 1] + dz[:, :, k])
+    p_int[:, :, -1] = ps / 2.0  # surface pressure bug
+    delp = p_int[:, :, 1:] - p_int[:, :, 0:-1]
+    return delp.rename({"ilev": "lev"}).rename(
+        "pressure_thickness_of_atmospheric_layer"
+    )
+
+
+def convert_to_fv3_format(
+    NE: int,
+    data_path: str,
+    file_name: str,
+    output_path: str,
+    output_file_name: str,
+    output_variables: List[str],
+):
+    """Converts a scream output file to the fv3gfs format.
+
+    Args:
+        NE: number of spectral elements per cubed face
+        data_path: path to input data directory, remote or local
+        file_name: either a single file name
+            or a wildcard pattern to match multiple names
+        output_path: path to output data directory, remote or local
+        output_file_name: name of output file
+        output_variables: list of variables to output
+    """
+    Ncols = NE ** 2 * 6 * NPG ** 2
+    x_dim = int(Ncols / 6 / NE / 3)
+    y_dim = int(Ncols / 6 / x_dim)
```

Comment:
I don't follow rationale for this definition of `x_dim` and `y_dim`. Let's chat offline.

Reply:
Also interested in this just for my own understanding.

Reply:
These were some magic integers I came up with, but per our discussion, no more reshaping. We use native scream output shape.